### ISSUE TITLE
[BUG] Error after updating from 1.6.1 to 1.6.2: `Cannot read properties of undefined (reading 'componentNodes')`

### ISSUE THREAD

**Issue by mateusluizfb** [FlowiseAI/Flowise#2024]
**Describe the bug**

I'm getting the following error after updating from 1.6.1 to 1.6.2. I'm using the docker image running in a EC2 instance:

```
TypeError: Cannot read properties of undefined (reading 'componentNodes')
    at /usr/local/lib/node_modules/flowise/dist/index.js:223:69
    at Layer.handle [as handle_request] (/usr/local/lib/node_modules/flowise/node_modules/express/lib/router/layer.js:95:5)
    at next (/usr/local/lib/node_modules/flowise/node_modules/express/lib/router/route.js:149:13)
    at Route.dispatch (/usr/local/lib/node_modules/flowise/node_modules/express/lib/router/route.js:119:3)
    at Layer.handle [as handle_request] (/usr/local/lib/node_modules/flowise/node_modules/express/lib/router/layer.js:95:5)
    at /usr/local/lib/node_modules/flowise/node_modules/express/lib/router/index.js:284:15
    at param (/usr/local/lib/node_modules/flowise/node_modules/express/lib/router/index.js:365:14)
    at param (/usr/local/lib/node_modules/flowise/node_modules/express/lib/router/index.js:376:14)
    at Function.process_params (/usr/local/lib/node_modules/flowise/node_modules/express/lib/router/index.js:421:3)
    at next (/usr/local/lib/node_modules/flowise/node_modules/express/lib/router/index.js:280:10)
```

docker-compose.yml:

```
version: "3.9"

services:
  flowise:
    image: flowiseai/flowise:1.6.2
    restart: always
    environment:
      - PORT=3001
      - FLOWISE_USERNAME=${FLOWISE_USERNAME}
      - FLOWISE_PASSWORD=${FLOWISE_PASSWORD}
      - DEBUG=${DEBUG}
      - DATABASE_TYPE=${DATABASE_TYPE}
      - DATABASE_PORT=${DATABASE_PORT}
      - DATABASE_HOST=${DATABASE_HOST}
      - DATABASE_NAME=${DATABASE_NAME}
      - DATABASE_USER=${DATABASE_USER}
      - DATABASE_PASSWORD=${DATABASE_PASSWORD}
      - FLOWISE_SECRETKEY_OVERWRITE=${FLOWISE_SECRETKEY_OVERWRITE}

    ports:
      - 3001:3001
    volumes:
      - ~/.flowise:/root/.flowise
    command: /bin/sh -c "sleep 3; flowise start"

```

I had to downgrade back to 1.6.1

**To Reproduce**

First make sure to have an EC2 instance running the flowise's docker image. Now in the instance's console:

1. From the 1.6.1, update the docker-compose version to 1.6.2 
3. Run docker-compose up with a clean build 

**Expected behavior**

Expected to update without problems, lots of features start to break due to the error described above.

**Screenshots**

<img width="1392" alt="Screenshot 2024-03-22 at 13 17 34" src="https://github.com/FlowiseAI/Flowise/assets/14284674/26b5fa5d-7c17-4b93-99b9-ea41a2443058">

**Flow**

All flows were breaking.

**Setup**

-   Installation docker, using docker-compose
-   Flowise Version 1.6.2
-   OS: EC2 Amazon Linux

**Comment by HenryHengZJ** [comment]
sorry for the bug, we jsut merged a fix [here](https://github.com/FlowiseAI/Flowise/pull/2021), can you temporarily downgrade to `1.6.1` first

**Comment by HenryHengZJ** [comment]
should be resolved in v`1.6.3`
